src/cookbooks/dau.md

```diff
@@ -93,3 +94,49 @@ GROUP BY
 ORDER BY
   submission_date_s3
 ```
+
+## Mobile Products
+
+The concept of usage is slightly different for mobile products compared to desktop Firefox.
+A single session of desktop Firefox is likely to span multiple days and we
+rely on a process to send a `main` ping once per day.
+A single session of a mobile application is likely to last only a few minutes and
+we have generally instrumented mobile applications to send a separate ping for
+each user session:
+
+* `core` pings are the canonical measure for usage on legacy mobile products
```

Comment:
Is it worthwhile to call out which specific products or is this hard to maintain?

Reply:
I will link here to another existing page that has a pretty good list of products.